src/Core/Domain/CartRule/Exception/CartRuleConstraintException.php

```diff
@@ -24,6 +24,8 @@
  * International Registered Trademark & Property of PrestaShop SA
  */
 
+declare(strict_types=1);
+
 namespace PrestaShop\PrestaShop\Core\Domain\CartRule\Exception;
```

Comment:
```suggestion
namespace PrestaShop\PrestaShop\Core\Domain\CartRule\Exception;
```

It's not a new file : don't add `declare(strict_types=1);`